src/main/java/org/spongepowered/common/mixin/core/entity/player/MixinEntityPlayerMP.java

```diff
@@ -897,4 +902,34 @@ public void setSpectatorTarget(@Nullable org.spongepowered.api.entity.Entity ent
         return event;
     }
 
+    @Override
+    public WorldBorder getWorldBorder() {
+        if (this.worldBorder == null) {
+            this.worldBorder = cloneBorder((WorldBorder) this.world.worldBorder);
```

Comment:
:-1:
Instead default to the `World`'s world border, like we do with scoreboards.